GMPWrapper/src/util/gmpUtil.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
```

Comment:
Why not put this in this folder?
https://github.com/key4hep/GMP/tree/master/GMPWrapper/GMPWrapper

Reply:
I'm ok with that, but I still prefer to have it in a separate util folder.

Reply:
Fine for me